webknossos-datastore/app/com/scalableminds/webknossos/datastore/controllers/BinaryDataController.scala

```diff
@@ -100,6 +102,7 @@ class BinaryDataController @Inject()(
       new ApiResponse(code = 400, message = "Operation could not be performed. See JSON body for more information.")
     ))
   def requestRawCuboid(
+      @ApiParam(value = "Datastore token identifying the requesting user", required = false) token: Option[String],
```

Comment:
I'm wondering if the `Option` part makes the parameter optional in swagger, too, or if the `required = False` is strictly necessary. Totally fine to leave as-is in any way.

Reply:
Turns out the default of required is False. I’ll remove it then.